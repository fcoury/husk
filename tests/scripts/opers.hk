let a = 1;
a += 8;
println(a);

a -= 3;
println(a);

a *= 2;
println(a);

a /= 2;
println(a);

a %= 2;
println(a);

a += 1;

a %= 2;
println(a);
