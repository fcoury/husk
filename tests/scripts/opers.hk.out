9
6
12
6
0
1
